build-docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#
+# prerequisites:
+#  * create the bxd namespace
+#  * create the docker-login secret in the bxd namespace
+#  * rabbit container running in rabbit-mq-ha.rabbit
+#
+#
+
+
+current_mvn_version=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec);
```

Comment:
```suggestion
current_coatrack_version=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec);
```

Reply:
it's not really the coatrack version, its the version in the pom, we also use the commit as version